src/otx/recipes/stages/segmentation/incremental.py

```diff
@@ -1,6 +1,6 @@
 _base_ = ["./train.py", "../_base_/models/segmentors/segmentor.py"]
 
-optimizer = dict(_delete_=True, type="Adam", lr=1e-3, eps=1e-08, weight_decay=0.0)
+optimizer = dict(_delete_=True, type="AdamW", lr=1e-3, eps=1e-08, weight_decay=0.0) # If default Adam is used, seg faults occurs
```

Comment:
Check later: This change requires performance verification when applied to `develop` branch.

Reply:
Yes but the good thing is `weight_decay` value is 0. It seems that Adam and AdamW are same if it's set to 0 referring torch docs [[AdamW]](https://pytorch.org/docs/stable/generated/torch.optim.AdamW.html), [[Adam]](https://pytorch.org/docs/stable/generated/torch.optim.Adam.html#torch.optim.Adam)